src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -60,21 +113,98 @@ export const PartnerDetail = () => {
                 <Skeleton width="75%" />
               )}
             </Typography>
-            {/* {partner ? (
+            <Tooltip title="Update Global Innovations Client">
               <IconButton
                 color="primary"
                 aria-label="edit partner"
-                onClick={editPartner}
+                onClick={() => editPartner('globalInnovationsClient')}
               >
                 <PencilCircledIcon />
               </IconButton>
-            ) : null} */}
+            </Tooltip>
+          </header>
+          <div className={classes.subheader}>
+            <Typography variant="h4">
+              {partner ? 'Partner Information' : <Skeleton width={200} />}
+            </Typography>
+            {partner && (
+              <Typography variant="body2" color="textSecondary">
+                Created {formatDate(partner.createdAt)}
```

Comment:
Use date time formatting for DateTime fields.

Reply:
Changed.  I did hear Amy say that we don't need to show the time on alot of these, if that becomes important we can either just use the date formatter for dateTime or we'll have to change the type